go.mod

```diff
@@ -34,8 +34,8 @@ require (
 	github.com/mattn/go-colorable v0.1.8 // indirect
 	github.com/mattn/go-isatty v0.0.13 // indirect
 	github.com/micromdm/scep/v2 v2.1.0
-	github.com/miekg/pkcs11 v1.0.3 // indirect
-	github.com/newrelic/go-agent v2.15.0+incompatible
+	github.com/miekg/pkcs11 v1.1.1 // indirect
+	github.com/newrelic/go-agent v2.16.4-0.20220510172711-aee135dc5dcf+incompatible
```

Comment:
It looks like the commit https://github.com/newrelic/go-agent/commit/aee135dc5dcf93df9dec7d55ab6ae6d40c0a4d7b is tagged with `v3.18.0`, can you use that instead?

Reply:
https://github.com/go-piv/piv-go/issues/105 Done.